### ISSUE TITLE
Academia link broken and pointing to 500 Error Page

### ISSUE THREAD

**Issue by dMLTquant** [OpenBB-finance/OpenBBTerminal#3662]
**What's the feature that should be improved?**
> A description of what's the feature you want improved, and a bit of context (why is that).

The website `Academia` Link does not work: `https://openbb.co/academia`

**Describe how you would like the feature improved**
> A description of what the current feature is vs what it would be after your suggestion.

- [ ] Option 1: Remove the link if the Academia feature no longer exists
- [ ] Option 2:  Fix the link to point to the correct location. 

**Possibly describe the ideal way to improve this**
> If you have thought about how you would do it, add it here.

NA

**Additional information**
Add any other information or screenshots about the feature improvement.

![image](https://user-images.githubusercontent.com/61799047/204743976-53e7d54a-6a2f-4eed-98d4-10df9f118c5c.png)

<img width="1338" alt="image" src="https://user-images.githubusercontent.com/61799047/204744102-8f2c11e7-bc4b-47f7-a4f3-3d7916878584.png">

**Comment by jose-donato** [comment]
Thanks for reporting this, on it :)

**Comment by jose-donato** [comment]
@dMLTquant can't reproduce this, can you tell me what browser are you using?

**Comment by dMLTquant** [comment]
> @dMLTquant can't reproduce this, can you tell me what browser are you using?

When I submitted the issue I was using Safari on MacOS. But I have just tested in on iOS with Safari and DuckDuckGo Browser and it still doesn’t work. Brave Browser fully breaks the app 🤷‍♂️🤦‍♂️

If I navigate to the actual page it does work. The link from the Home Page footer looks like is the problem. 

Hope this helps somehow identify the problem 

![C5A18148-7A5A-4E0E-B15B-3906CE9498B9](https://user-images.githubusercontent.com/61799047/204880483-d4f87821-14dc-42a8-9d0c-ff51803ab2e8.png)

![09B9E519-9285-491C-A284-15D231586FE0](https://user-images.githubusercontent.com/61799047/204880534-a78f7926-be31-470a-be4d-9adf9f65e76e.jpeg)
